content/docs/auth/quick-start/nextjs-api-only.md

```diff
@@ -0,0 +1,445 @@
+
+  </LeftContent>
+  <RightCode label="app/api/auth/[...path]/route.ts">
+
+```typescript
+import { authApiHandler } from '@neondatabase/neon-js/auth/next/server';
+
+export const { GET, POST } = authApiHandler();
+```
+
+  </RightCode>
+</TwoColumnStep>
+
+<TwoColumnStep title="Add neonAuthMiddleware()">
+  <LeftContent>
+
+The `neonAuthMiddleware()` ensures that user is authenticated before the request reaches your page components or API routes. Create `proxy.ts` file in your project root:
+
+  </LeftContent>
+  <RightCode label="proxy.ts">
+
+```typescript
+import { neonAuthMiddleware } from "@neondatabase/neon-js/auth/next/server";
+
+export default neonAuthMiddleware({
+  // Redirects unauthenticated users to sign-in page
+  loginUrl: "/auth/sign-in",
+});
+
+export const config = {
+  matcher: [
+    // Protected routes requiring authentication
+    "/account/:path*",
+  ],
+};
+```
+
+  </RightCode>
+</TwoColumnStep>
+
+
+
+<Admonition type="note">
+Your Next.js project is now fully configured to use Neon Auth. Now, lets proceed with setting up the Auth UI Provider and wrap your layout with auth context. 
+</Admonition>
+
+
+<TwoColumnStep title="Configure the auth clients">
+  <LeftContent>
+
+**Client Components:**
+  - The Auth UI components are client rendered and need access to the auth APIs. Lets first create the auth client in `lib/auth/client.ts` file then we pass it to `NeonAuthUIProvider`
+
+**Server Components:** 
+  - To use Auth APIs in server components and server actions, you can also create auth-server in `lib/auth/server.ts` file.
+
+  </LeftContent>
+  <RightCode>
+<Tabs labels={["Auth Client", "Auth Server"]}>
+
+<TabItem>
+
+Copy and paste following code in `lib/auth/client.ts` file:
+
+```tsx
+'use client';
+
+import { createAuthClient } from '@neondatabase/neon-js/auth/next';
+
+export const authClient = createAuthClient();
+```
+
+  </TabItem>
+  <TabItem>
+
+Copy and paste following code in `lib/auth/server.ts` file:
+
+```tsx
+import { createAuthServer } from '@neondatabase/neon-js/auth/next/server';
+
+export const authServer = createAuthServer();
+```
+
+  </TabItem>
+  </Tabs>
+  </RightCode>
+</TwoColumnStep>
+
+<TwoColumnStep title="Create Sign up form">
+  <LeftContent>
+
+Lets create a sign-up form and action in `app/auth/sign-up/page.tsx` and `app/auth/sign-up/actions.ts` files respectively using the auth server instance we created in previous step
+
+  - To create user with email and password, we will use `authServer.signUp.email()` with user name, email address, and password
+  - You can optionally add business logic before invoking the API, for example restrict signups to emails ending with `@my-company.com`
+
+  </LeftContent>
+  <RightCode>
+
+  <Tabs labels={["Signup action", "Signup form"]}>
+  <TabItem>
```

Comment:
Be clearer on which tab is `app/auth/sign-up/page.tsx` and which is `app/auth/sign-up/actions.ts`?